src/cov.jl

```diff
@@ -29,6 +29,21 @@ scattermat_zm(x::DenseMatrix, wv::WeightVec, vardim::Int) =
     _symmetrize!(Base.unscaled_covzm(x, _scalevars(x, values(wv), vardim), vardim))
 
 if VERSION < v"0.5.0-dev+679"
+"""
+    scattermat(x, [wv,] [mean=nothing,] [vardim=1]) -> Matrix
+
+Compute the scatter matrix, optionally specifying a weighting vector.
+The function accepts two optional keyword arguments:
+
+  • `mean`: a known mean value. By default it is set to `nothing`,
+    which indicates that the mean is unknown, and the function will
+    compute the mean. Specifying `mean=0` indicates that the data
```

Comment:
I'm not sure the last sentence is very useful: by definition, centered data has zero mean, and subtracting zero has no effect. You're actually documenting an implementation detail which AFAIK has no user-visible effect.